turf/rhumb_bearing/_rhumb_bearing.py

```diff
@@ -0,0 +1,62 @@
+from math import fmod
+from typing import Dict, List, Union
+
+import numpy as np
+
+from turf.helpers import degrees_to_radians, radians_to_degrees
+from turf.helpers import Point
+from turf.invariant import get_coord
+
+
+def rhumb_bearing(
+    origin: Union[List, Point], destination: Union[List, Point], options: Dict = None
+) -> float:
+    """
+    Takes two {Point|points} and finds the bearing angle between them along a
+    Rhumb line
+     * i.e. the angle measured in degrees start the north line (0 degrees)
+
+    # https://en.wikipedia.org/wiki/Rhumb_line
+
+    :param start: starting point [lng, lat] or Point feature
+    :param end: ending point [lng, lat] or Point feature
+    :param options: Optional parameters
+        :param final: calculates the final bearing if True
```

Comment:
here should maybe be `options["final"]: Calculates the final bearing if True`

Reply:
👍